### PR TITLE
Add notes to README about support for cloud-based Bitbucket and Jira

### PR DESCRIPTION
**Pull request by rybak**
Details in the message of the second commit.

I would like to get some feedback on the wording of the new paragraphs in the README, hence a pull request, rather than a direct commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +144,9 @@ This user script changes favicons of Jira pages (tab icon in browsers) to the
 avatar of individual Jira projects, which makes it easier to distinguish browser
 tabs of tickets in different projects.
 
+This scripts supports both Jira Server (self-hosted) and Jira Software
```
Comment (dtruebin) [inline README.md:147]: Jira _Software_ is the "usual" (i.e., non-service desk) flavour of Jira. Exists both in Server and Cloud options.

**Review by dtruebin — APPROVED** [review]
4c76c8f msg has a typo: "bitbucke**r**.org" 😛

File: README.md (hunk 1 above)
Comment (rybak) [reply README.md:147]: So it should just say:

> This scripts supports both Jira Server (self-hosted) and Jira Cloud.

I guess?

File: README.md [hunk 2]
```diff
@@ -55,6 +59,8 @@ Some instances of Bitbucket are deployed with custom look and feels, which have
 unfortunate color of the header.  This user style allows changing the color of
 Bitbucket header to any color.
 
+This style supports both Bitbucket Server and Bitbucket cloud.
```
Comment (rybak) [inline README.md:62]: Oops, "Cloud" isn't capitalized in two places:

```suggestion
This style supports both Bitbucket Server and Bitbucket Cloud.
```

File: README.md [hunk 3]
```diff
@@ -64,6 +70,8 @@ This user script changes favicons of Bitbucket pull request pages (tab icon in
 browsers) to the avatar of pull request's author, which makes it easier to
 distinguish browser tabs.
 
+This script supports both Bitbucket Server and Bitbucket cloud.
```
Comment (rybak) [inline README.md:73]: Oops, "Cloud" isn't capitalized in two places:

```suggestion
This script supports both Bitbucket Server and Bitbucket Cloud.
```

File: README.md (hunk 1 above)
Comment (rybak) [reply README.md:147]: And fix the incorrect plural `scripts` too.

Comment (rybak) [reply README.md]: Fixed in [0b182de](https://github.com/rybak/atlassian-tweaks/pull/8/commits/0b182debf7ce273851f1e70681f21fa16154624d).

**Comment by rybak** [comment]
Force pushed to squash fixup commits. Rebased on `main`. Range diff:

```range-diff
$ git range-diff 0b182de...fa71737
-:  ------- > 1:  01ba342 new style bitbucket-fast-and-furious
1:  aa9b8ab ! 2:  d7cace2 README: clarify mention of Bitbucket Server
    @@ Metadata
     Author: Andrei Rybak <rybak.a.v@gmail.com>

      ## Commit message ##
    -    README: clarify mention of Bitbucket Server
    +    README: mention Bitbucket Server version 7.6 explicitly

         Nowadays, Atlassian has many flavors of Bitbucket:

    @@ Commit message
           - Bitbucket Data Center
           - Bitbucket Cloud

    -    To avoid confusion, refer to Bitbucket Server explicitly in README.md.
    +    To avoid confusion, refer to Bitbucket Server version 7.6 explicitly in
    +    README.md.

      ## README.md ##
     @@ README.md: ### Bitbucket – fix selection of breadcrumbs
2:  4c76c8f ! 3:  fa71737 README: mark scripts & styles for support of cloud
    @@ README.md: ### Bitbucket – header color
      unfortunate color of the header.  This user style allows changing the color of
      Bitbucket header to any color.

    -+This style supports both Bitbucket Server and Bitbucket cloud.
    ++This style supports both Bitbucket Server and Bitbucket Cloud.
     +
      https://github.com/rybak/atlassian-tweaks/raw/main/bitbucket-header-color.user.css

    @@ README.md: ### Bitbucket – PR author avatar as favicon
      browsers) to the avatar of pull request's author, which makes it easier to
      distinguish browser tabs.

    -+This script supports both Bitbucket Server and Bitbucket cloud.
    ++This script supports both Bitbucket Server and Bitbucket Cloud.
     +
      https://github.com/rybak/atlassian-tweaks/raw/main/bitbucket-pull-request-avatars.user.js

    @@ README.md: ### Jira – project avatar as favicon
      avatar of individual Jira projects, which makes it easier to distinguish browser
      tabs of tickets in different projects.

    -+This scripts supports both Jira Server (self-hosted) and Jira Software
    -+(cloud-based).
    ++This script supports both Jira Server (self-hosted) and Jira Cloud.
     +
      https://github.com/rybak/atlassian-tweaks/raw/main/jira-project-avatar-as-favicon.user.js

3:  d773d33 < -:  ------- fixup! README: mark scripts & styles for support of cloud
4:  0b182de < -:  ------- fixup! README: mark scripts & styles for support of cloud
```

**Pull request merged**